sdk/servicebus/Azure.Messaging.ServiceBus/tests/Client/ServiceBusClientLiveTests.cs

```diff
@@ -84,13 +84,13 @@ await using (var client = new ServiceBusClient(TestEnvironment.FullyQualifiedNam
         /// </summary>
         ///
         [Test]
-        public async Task ClientCanConnectUsingSharedKeyCredentialWithSharedKey()
+        public async Task ClientCanConnectUsingSharedKeyCredential()
         {
             await using (var scope = await ServiceBusScope.CreateWithQueue(enablePartitioning: true, enableSession: false))
             {
                 var options = new ServiceBusClientOptions();
                 var audience = ServiceBusConnection.BuildConnectionResource(options.TransportType, TestEnvironment.FullyQualifiedNamespace, scope.QueueName);
-                var credential = new ServiceBusSharedAccessKeyCredential(TestEnvironment.SharedAccessKeyName, TestEnvironment.SharedAccessKey);
+                var credential = new AzureNamedKeyCredential(TestEnvironment.SharedAccessKeyName, TestEnvironment.SharedAccessKey);
```

Comment:
Would it be possible to have a live test where we actually update the key after the client is already constructed and after we've already done some operations with the initial key? Maybe we could just update it to use the secondary key from the namespace.

Reply:
We validate this in the `SharedAccessTokenCredentialTests`.   I don't see enough value in doing this as a Live to run the additional infrastructure through the test resources to query and inject a second key.